LibraBFT/Impl/Properties/Util.agda

```diff
@@ -69,7 +66,7 @@ module OutputProps where
     NoMsgs⇒NoVotes = proj₁ ∘ proj₂ ∘ NoMsgs⇒×
 
     OneVote : VoteMsg → List Author → Set
-    OneVote vm pids = List-filter isSendVote? outs ≡ (SendVote vm pids ∷ [])
+    OneVote vm pids = List-filter isOutputMsg? outs ≡ (SendVote vm pids ∷ [])
```

Comment:
@haroldcarr Is it the case that when a peer discovers that it needs to advance to a new epoch while processing a proposal, it will never generate a vote for that proposal?